bool fmeasure_request
---
float32 fmeasure